src/rules/value-list-max-empty-lines/index.js

```diff
@@ -0,0 +1,60 @@
+import {
+  isNumber,
+  repeat,
+} from "lodash"
+import {
+  isStandardSyntaxDeclaration,
+  isStandardSyntaxProperty,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+import styleSearch from "style-search"
+
+export const ruleName = "value-list-max-empty-lines"
+
+export const messages = ruleMessages(ruleName, {
+  expected: max => `Expected no more than ${max} empty line(s)`,
+})
+
+export default function (max) {
+  const maxAdjacentNewlines = max + 1
+
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, {
+      actual: max,
+      possible: isNumber,
+    })
+    if (!validOptions) { return }
+
+    root.walkDecls(decl => {
+      if (
+        !isStandardSyntaxDeclaration(decl)
+        || !isStandardSyntaxProperty(decl.prop)
```

Comment:
Why are these checks needed? Is there anything about non-standard syntax declarations or props that would trip up this rule? If not, let's check them like everything else.